cssnano.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	preset: [
+		'default',
+		{ discardComments: false }
```

Comment:
Just to be clear (if that's what it's about indeed):

```suggestion
		// The CSS comment is needed for the theme headers.
		{ discardComments: false }
```